.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+Delete as appropriate
+
+# If the issue a bug:
+
+## Screenshot of bug:
+
+...
+
+## Explanation of bug:
+
+...
+
+## Steps to reproduce:
+
+...
+
+# If the issue is enhancement suggestion:
+
+## What the enhancement is
+
+...
+
+## Why it could be helpful
+
+...
+
+PLEASE DON'T FORGET TO APPROPRIATELY TAG THE ISSUE AS A BUG OR ENHANCEMENT!
```

Comment:
@moosichu are you talking about labels here? AFAIK, labels can only be added by those with write access

Reply:
OK, I will remove references to that then

Reply:
Done